recipes/msys2/all/conanfile.py

```diff
@@ -24,6 +24,8 @@ class MSYS2Conan(ConanFile):
     def configure(self):
         if self.settings.os_build != "Windows":
             raise ConanInvalidConfiguration("Only Windows supported")
+        if self.settings.arch_build == "x86" and self.version >= version("20201109"):
```

Comment:
Are we ready to use `arch/os` instead of `arch/os`?

Reply:
will try